src/ssl.c

```diff
@@ -16060,7 +16060,7 @@ const char* wolfSSL_CIPHER_get_name(const WOLFSSL_CIPHER* cipher)
         return NULL;
     }
 
-    #ifndef WOLFSSL_CIPHER_INTERNALNAME
+    #if !defined(WOLFSSL_CIPHER_INTERNALNAME) && !defined(NO_ERROR_STRINGS)
```

Comment:
This and the change on line 16077 are good however the test case needs to be updated for new functionality. See tests/api.c line 2021